resources/monitoring/Chart.yaml

```diff
@@ -1,4 +1,7 @@
 apiVersion: v1
+#This value is used as serviceMonitorSelector matching label.
```

Comment:
IMO in future, they should change that behavior and do not connect it with the chart name. Could u ping them?

Reply:
right, but I'm just fixing bug here. Normally I agree that should be in values.yaml
owners of the component are included in review